data/sheragi/archaeology missions.txt

```diff
@@ -1280,6 +1280,13 @@ mission "Sheragi Archaeology: The Emerald Sword 2"
 			label return
 			`	"What do I get in return?" you ask.`
 			`	"Fame and fortune, my friend. And if the ship is still there, I think you deserve to keep it," Albert responds. "You've helped me so much, and you went to hell and back to get this black box decoded. Give us some time to assemble a crew to survey the system and help refurbish the ship, and in the meantime, go get us a jump drive. Come back to us once you have it."`
+			choice
+				`	"You're just going to let me have it? Isn't this something of a priceless historical artifact?"`
+					goto askkeepship
+				`	"Thank you."`
+					accept
+			label askkeepship
+			`	Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed or stolen in a pirate raid. The militia works hard, but they can't be everywhere."`
```

Comment:
```suggestion
			`	Albert responds, "Well, yes, but I'd feel much more comfortable with it in your hands than sitting here on <origin> waiting to be destroyed in loose fire during a pirate raid. Besides that though, while the Free Worlds work hard and have a stronger presence here than the Navy ever did, they can't stop everyone who may have an interest in this ship. At least with you it's a moving target."`
```

Can't you only get this far after completing the main plot?

Even if it is still just the militia though, they don't need to be everywhere to stop this being stolen, just here, so phrasing it that way reads as a cop out to me.

Especially when the real threat to a piece in a guarded museum is most likely not random pirates grabbing whatever they can, but organized attack from specific groups (some of whom also either are or are working with pirates [BBub and Mammon spring readily to mind]) deliberately targeting the ship and/or its tech specifically.

Suggested change also leaves a bit more of an explicit hook that can be capitalized on later if we desire.

Reply:
A previous mission `Sheragi Archaeology: The Box 1` requires `has "main plot completed"`, so I feel safe mentioning the FW militia directly.
Your suggestion is a definitely improvement to my eyes.  Thank you!